gemrb/plugins/TLKImporter/TLKImporter.cpp

```diff
@@ -364,7 +364,7 @@ String TLKImporter::GetString(ieStrRef strref, STRING_FLAGS flags)
 		str->ReadDword(StrOffset);
 		str->ReadDword(l);
 
-		if (type & 1) {
+		if (type & 1 && l > 0) {
```

Comment:
Doesn't this potentially cause similar problems in ResolveTags later, since it's accessing [0] of an empty string? It would be cleaner to just immediately return if l == 0, since the rest of the function can't do anything useful any more.

Reply:
Yes, that looks bad on its own, but there is only this use here. Then I'll make it drop out with an empty string instead.